pkg/helm/v3/converter.go

```diff
@@ -0,0 +1,62 @@
+package v3
+
+import (
+	"github.com/helm/helm-2to3/pkg/common"
+	helm2to3v2 "github.com/helm/helm-2to3/pkg/v2"
+	helm2to3v3 "github.com/helm/helm-2to3/pkg/v3"
+)
+
+// Converter Converts a given helm 2 release with all its release versions to helm 3 format and deletes the old release from tiller
+type Converter struct {
+	TillerNamespace string
+	KubeConfig      string // file path to kubeconfig
+}
+
+// V2ReleaseExists helps you check if a helm v2 release exists or not
+func (c Converter) V2ReleaseExists(releaseName string) bool {
+	retrieveOpts := helm2to3v2.RetrieveOptions{
+		ReleaseName:     releaseName,
+		TillerNamespace: c.TillerNamespace,
+	}
+	kubeConfig := common.KubeConfig{
+		File: c.KubeConfig,
+	}
+	v2Releases, err := helm2to3v2.GetReleaseVersions(retrieveOpts, kubeConfig)
+	if err != nil {
+		return false
+	}
```

Comment:
I do not think we can safely swallow this error, instead I would return a `(bool, error)` and bail the release process on errors.